xbmc/guilib/GUITextBox.cpp

```diff
@@ -293,6 +296,12 @@ bool CGUITextBox::OnMessage(CGUIMessage& message)
         return true;
       }
     }
+
+    if (message.GetMessage() == GUI_MSG_SET_TYPE)
+    {
+      UseMonoFont(message.GetParam1() == 1 ? true : false);
```

Comment:
remove ternary operation because `message.GetParam1() == 1` should be enough

Reply:
It will warn on certain compilers.

Reply:
okay